components/automate-chef-io/content/docs/iam-v2-overview.md

```diff
@@ -152,23 +151,102 @@ The default policies are: Viewers, Editors, Administrator, and Ingest.
+   With no selection (the default), you are thereby saying you want *no* filtering, hence displaying resources for *all* projects for which you have permission.
+
+Note that out-of-the-box we provide a *project-owner* role so the global admin may delegate much of these responsibilities for project management to others, to alleviate some of the burden for the global admin.
+
+### Properties of a Project
+
+A v2.1 project consists of a list of **rules**, where each rule describes a group of node characteristics.
+
+Property   | Description
+-----------|------------
+Rules      | list of tagging rules
+
+A rule consists of a list of **conditions**, where each condition describes a single characteristic.
+It also contains a type where <<TODO describe what node and event mean>>
+
+Property   | Description
+-----------|------------
+Type       | one of (node, event)
+Conditions | list of node characteristics
+
+A condition consists of these properties:
+
+Property   | Description
+-----------|------------
+Attribute  | for **event** types: one of (CHEF_ORGANIZATION, CHEF_SERVER)
+Attribute  | for **node** types: one of (CHEF_ORGANIZATION, CHEF_SERVER, ENVIRONMENT, CHEF_ROLE, CHEF_TAG, CHEF_POLICY_NAME, CHEF_POLICY_GROUP)
+Operator   | one of (equals, member_of)
+Values     | list of one or more values to match on the specified attribute
+
+### Tagging Resources
+
+There are two broad categories of resources that may be tagged: your set of nodes feeding data into Chef Automate and the set of IAM resources themselves.
+Any of these that you do not explicitly tag (as described next) are considered *unassigned* with respect to projects.
+We denote this group of unassigned projects as being in the `(unassigned)` project, including the parentheses as a way to connote that it is a special designator.
+At the start, then, all of your resources are not tagged with any projects and thus can be viewed by selecting either `All Projects` or just the `(unassigned)` project in the global project filter.
+
+Taggable IAM resources include teams, tokens, policies, and roles. Teams and tokens may be tagged with projects directly in the UI; policies and roles still can only be tagged on the command-line (since you cannot create or modify those at all in the UI yet).
+
+Tagging your set of nodes is done via the rules you create for the project, mentioned just briefly above.
+A rule specifies one or more conditions, and each condition specifies a set of attribute values that a node must satisfy to be tagged with the given project.
+Once you define your set of projects with their contained rules and conditions, you then use the **Update Projects** button on the main projects list page to apply those definitions to all your nodes.
+
+## Policies, Roles, and Projects in the UI
+
+IAM v2.1 introduces partial policy management from within Chef Automate in your browser.
```

Comment:
whats this partial about? can we just say you can manage members from the ui, instead of indicating its not complete or something because the editing of a policy from the ui will not happen until after ga.